docs/modules/ROOT/pages/how-tos/creating.adoc

```diff
@@ -1 +1,127 @@
-= Creating applications and components
\ No newline at end of file
+= Creating applications and components
+
+To create an application in {ProductName}, someone must enable build pipelines in your instance of {ProductName}. This includes creating a GitHub application, so {ProductName} can access users' pull requests, and configuring a smee channel, to listen for those PRs. The first procedure in this document explains how to perform these tasks. 
+
+Once build pipelines are enabled, users can create their applications in {ProductName}. The second procedure below describes how to create an application. 
+
+== Enabling build pipelines for your instance of {ProductName}:
+
+.Prerequisites:
+
+* Install link:https://github.com/konflux-ci/konflux-ci/tree/main[{ProductName}]
+* Read/write access to your {ProductName} instance’s repository
```

Comment:
This is not entirely clear to me. 
The user needs to have a local copy of the konflux-ci repository and they need read/write access on their local copy.

I think the chances for someone to clone the repository to their workstation and make it without read/write access to themselves are slim. For the rest, I think this line might be confusing.

Reply:
ok. I understand better the scenario now.